addons/binding/org.openhab.binding.km200/ESH-INF/thing/device.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="km200"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+   <bridge-type id="kmdevice">
+        <label>KM200/100/50</label>
+        <description>The KM200 binding is communicating with a Buderus Logamatic web KM200 / KM100 / KM50. It is possible to receive and send parameters like string or float values.</description>
+        
+         <config-description>
+	        <parameter name="ip4Address" type="text" required="true">
+	            <label>IP4 address</label>
+	            <context>network-address</context>
+	            <description>IP4 Address of the KMXXX device</description>
+	        </parameter>
+	        <parameter name="privateKey" type="text" required="false">
+	           <label>Private key</label>
```

Comment:
Should this also have the `password` context, so UIs will hide the actual value?

Reply:
It would be possible, but this key is realy long so I didn't activate it.